src/Console/Command/FixCommand.php

```diff
@@ -108,19 +115,43 @@ protected function configure()
             ->setDefinition(
                 array(
                     new InputArgument('path', InputArgument::OPTIONAL, 'The path', null),
-                    new InputOption('allow-risky', '', InputOption::VALUE_REQUIRED, 'Are risky fixers allowed (can be yes or no)', null),
+                    new InputOption(
+                        'allow-risky',
+                        '',
+                        InputOption::VALUE_REQUIRED,
+                        'Are risky fixers allowed (can be yes or no)',
+                        null
+                    ),
                     new InputOption('config', '', InputOption::VALUE_REQUIRED, 'The path to a .php_cs file ', null),
-                    new InputOption('dry-run', '', InputOption::VALUE_NONE, 'Only shows which files would have been modified'),
+                    new InputOption(
+                        'dry-run',
+                        '',
+                        InputOption::VALUE_NONE,
+                        'Only shows which files would have been modified'
+                    ),
                     new InputOption('rules', '', InputOption::VALUE_REQUIRED, 'The rules', null),
-                    new InputOption('using-cache', '', InputOption::VALUE_REQUIRED, 'Does cache should be used (can be yes or no)', null),
+                    new InputOption(
+                        'using-cache',
+                        '',
+                        InputOption::VALUE_REQUIRED,
+                        'Does cache should be used (can be yes or no)',
+                        null
+                    ),
                     new InputOption('cache-file', '', InputOption::VALUE_REQUIRED, 'The path to the cache file'),
                     new InputOption('diff', '', InputOption::VALUE_NONE, 'Also produce diff for each file'),
-                    new InputOption('format', '', InputOption::VALUE_REQUIRED, 'To output results in other formats', 'txt'),
+                    new InputOption(
```

Comment:
:-1: for inconsistency for having InputOption sometimes as one-liners and sometimes not.